base_layer/mmr/src/balanced_binary_merkle_proof.rs

```diff
@@ -400,7 +400,7 @@ mod test {
     fn test_real_world_example() {
         hash_domain!(
             ValidatorNodeBmtHashDomain,
-            "com.tari.tari_project.base_layer.core.validator_node_mmr",
+            "com.tari.test.base_layer.core.validator_node_mmr",
```

Comment:
This change fails the test. It may be the case that the domain _should_ collide with the production domain, but the intent of the test and source of the test vectors is unclear. Would appreciate any insights from @sdbondi, who authored the test in #5432;

Reply:
I'm told by @sdbondi that the test can be removed.